db/migrate/20230106103345_modifie_mode_tutoriel_dans_compte.rb

```diff
@@ -0,0 +1,8 @@
+class ModifieModeTutorielDansCompte < ActiveRecord::Migration[7.0]
+  def up
+    Structure.activees.find_each do |structure|
+      comptes = Compte.where(structure: structure).validation_acceptee
+      comptes.update_all(mode_tutoriel: false)
+    end
```

Comment:
La migration peut être effectué plus rapidement de la manière suivante : 

```suggestion
  structure_ids = Structure.activees.select(:id)
  comptes = Compte.where(structure_id: structure_ids).validation_acceptee
  comptes.update_all(mode_tutoriel: false)
```

cela ne génèrera qu'un seule requête SQL au global